src/components/results/feedback/modal/question.js

```diff
@@ -0,0 +1,99 @@
+import PropTypes from "prop-types";
+import {useState, useEffect} from "react";
+import style from "./style.scss";
+
+function MultipleChoice({onChange, questionId, questionText, options}) {
+  const [answer, setAnswer] = useState("");
+  useEffect(() => {
+    onChange({
+      question_type: "Multiple Choice",
+      question_id: questionId,
+      selected_option_id: answer
+    });
+  }, [answer]);
+  return (
```

Comment:
We like to add a line of white space around `useEffect` and/or before the JSX return statement

Reply:
updated